.github/workflows/integration-tests.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         path: _release/stack-*
 
   github-release:
-    name: Create Github release
+    name: Create GitHub release
```

Comment:
Brand.

Note that I'm happy to drop any particular changes... (I'll tag this later in code, since I'm less comfortable w/ Haskell)